fabrics.c

```diff
@@ -1317,12 +1324,6 @@ int nvmf_config(const char *desc, int argc, char **argv)
 	if (dump_config)
 		nvme_dump_config(r);
 
-out:
-	if (hid)
-		free(hid);
-	if (hnqn)
-		free(hnqn);
-	nvme_free_tree(r);
 	return -errno;
```

Comment:
This could probably be `return 0` now

Reply:
I'm not following. `hid` and `hnqn` aren't used anywhere and their values match `hostid` and `hostnqn`. So why not remove `hid` and `hnqn` and instead mark `hostid` and `hostnqn` as `_cleanup_free_`?